data/json/items/ammo_types.json

```diff
@@ -430,6 +430,12 @@
         "id" : "unfinished_char",
         "name" : "semi-charred fuel",
         "default" : "unfinished_charcoal"
+    },
+    {
+        "type" : "ammunition_type",
+        "id" : "gas_fungicidal",
+        "name" : "fungicidal gas",
+        "default" : "gas_fungicidal"
```

Comment:
I was having trouble with assigning a more general chemical_spray ammo-type. I wanted something like this:
```{
"type" : "ammunition_type",
"id" : "chemical_spray",
"name" : "sprayable chemical",
"default" : "gas_fungicidal"
}```